tt_metal/impl/device/device.cpp

```diff
@@ -1608,9 +1608,9 @@ bool Device::close() {
         for (auto it = not_done_dispatch_cores.begin(); it != not_done_dispatch_cores.end(); it++) {
             const auto &phys_core = *it;
             if(llrt::is_ethernet_core(phys_core, this->id_)) {
-                log_info(tt::LogMetal, "Ethernet dispatch core {} on Device {} is idle. Closing Device {}", phys_core.str(), mmio_device_id, this->id());
+                log_debug(tt::LogMetal, "Ethernet dispatch core {} on Device {} is idle. Closing Device {}", phys_core.str(), mmio_device_id, this->id());
             } else {
-                log_info(tt::LogMetal, "Resetting core {} on Device {} when closing Device {}", phys_core.str(), mmio_device_id, this->id());
+                log_debug(tt::LogMetal, "Resetting core {} on Device {} when closing Device {}", phys_core.str(), mmio_device_id, this->id());
```

Comment:
ok for this PR, but in general it is good to keep these cleanups in separate commits in case regressions are introduced

Reply:
I'll make two commits. I think there's non-zero chance there are some regressions 🙃